### ISSUE TITLE
When a facet limit area starts out open, it has wrong aria-expanded

### ISSUE THREAD

**Issue by jrochkind** [projectblacklight/blacklight#2194]
If a sidebar facet area already has a limit selected from it relevant to the current search, it begins expanded, rather than collapsed. 

But it's aria-expanded attribute is still wrongly "false". 

The relevant code is at https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_facet_layout.html.erb#L3-L8 plus line 12. (thanks to @cbeer for the tip)

The code seems to be trying to make an element initially expanded by: 

*  _refraining_ from adding "collapse" CSS class to trigger button as well as content block. 

This does not actually seem to match the examples in [Bootstrap 4 collapsible docs](https://getbootstrap.com/docs/4.3/components/collapse/), which don't seem to put a `collapse` class on the trigger button in any event. 

The Bootstrap docs don't provide a very clear example of "collapsible but initially expanded", but... aha, they seem to say you should not remove the `collapse` class from the content, but instead should add a `show` class, and should _also_ set aria-expanded properly. I don't believe the `collapse` class on the trigger button itself is appropriate. 

>  If the collapsible element is closed by default, the attribute on the control element should have a value of aria-expanded="false". If you’ve set the collapsible element to be open by default using the show class, set aria-expanded="true" on the control instead.
